.github/workflows/publishpackage.yml

```diff
@@ -1,12 +1,26 @@
 name: Publish Python Package
-
-on: 
+on:
   push:
     branches:
       - master
-
 jobs:
+  test:
+    runs-on: ubuntu-18.04
+    name: Run Tests/Linters
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Install
+        run: |
+          curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python3
```

Comment:
We should be specifying which version(s) of python we want to test against instead of just using whatever python happens to be around. See [here](https://help.github.com/en/actions/language-and-framework-guides/using-python-with-github-actions) for details on how to set that up.

Reply:
Neat; testing against 3.7 & 3.8 now.